melodic/distribution.yaml

```diff
@@ -6269,6 +6269,16 @@ repositories:
       url: https://github.com/ros-drivers/odva_ethernetip.git
       version: indigo-devel
     status: unmaintained
+  omnibase:
+    doc:
+      type: git
+      url: https://github.com/ERC-BPGC/omnibase.git
+      version: master
+    source:
+      type: git
+      url: https://github.com/ERC-BPGC/omnibase.git
```

Comment:
The package.xml for all of the packages in that source repository say that the license is "ERC", but there is no text of exactly what that license is.

If this is going to be open source, we highly recommend using one of the standard licenses (BSD or Apache 2).  We are not lawyers, so we aren't qualified to review new licenses and determine whether they give us the ability to redistribute the software.

Reply:
I will change the license to BSD then, the packages are going to be open source. This is my first time in this matter. 
Thanks for the suggestions!

Reply:
I have changed the licenses for all the packages.